crates/store/src/db/sql/mod.rs

```diff
@@ -582,17 +584,22 @@ pub fn insert_nullifiers_for_block(
     nullifiers: &[Nullifier],
     block_num: BlockNumber,
 ) -> Result<usize> {
+    let serialized_nullifiers: Vec<Value> =
+        nullifiers.iter().map(Nullifier::to_bytes).map(Into::into).collect();
+    let serialized_nullifiers = Rc::new(serialized_nullifiers);
+
+    let mut stmt = transaction
+        .prepare_cached("UPDATE notes SET consumed = TRUE WHERE nullifier IN rarray(?1)")?;
+    stmt.execute(params![serialized_nullifiers])?;
```

Comment:
I'm unsure if lumping in the nullifier set here is a good idea or not. We're now updating `nullifers` and `notes` table here. 

The benefit is that we can re-use the nullifier serialization.